torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -542,18 +542,18 @@ class IrParser {
             }
 
             // NOLINTNEXTLINE(cppcoreguidelines-avoid-magic-numbers)
-            auto training = constant_as<bool>(node->input(5));
-            TORCH_INTERNAL_ASSERT(
-                training.has_value(),
-                "The training (bool) parameter is required.");
-            const bool kTraining = training.value();
+            // auto training = constant_as<bool>(node->input(5));
```

Comment:
why keep commented out code? if it's not needed we should delete it (even if we anticipate future uses, we can add it back when we need it)

Reply:
@rdspring1 wants to keep it there.